sidecar.sh

```diff
@@ -23,40 +23,36 @@ populate_notebook() {
         filesize=$(echo $j | jq -r .file_size)
 
         # Need to add a literal newline character that's why the quote is ending on next line
-        drs_pull="!gen3 drs-pull object $obj
-
-"
+        drs_pull="!gen3 drs-pull object $obj"
         # Need to add a literal newline character that's why the quote is ending on next line
-        jq --arg cmd "# File name: $filename - File size: $filesize
-" '.cells[5].source += [$cmd]' "$FOLDER/data.ipynb" > "$FOLDER/data.tmp"
+        jq --arg cmd "# File name: $filename - File size: $filesize" '.cells[5].source += [$cmd]' "$FOLDER/data.ipynb" > "$FOLDER/data.tmp"
         mv "$FOLDER/data.tmp" "$FOLDER/data.ipynb"
 
         jq --arg cmd "$drs_pull" '.cells[5].source += [$cmd]' "$FOLDER/data.ipynb" > "$FOLDER/data.tmp"
         mv "$FOLDER/data.tmp" "$FOLDER/data.ipynb"
 
-
     done
     log "Done populating notebook"
 }
 
 function populate() {
     log "querying manifest service at $GEN3_ENDPOINT/manifests"
-    MANIFEST=$(curl -s -H "Authorization: Bearer ${ACCESS_TOKEN}" "https://$GEN3_ENDPOINT/manifests")
+    MANIFESTS=$(curl -s -H "Authorization: Bearer ${ACCESS_TOKEN}" "https://$GEN3_ENDPOINT/manifests/")
     log "querying manifest service at $GEN3_ENDPOINT/metadata"
     METADATA=$(curl -s -H "Authorization: Bearer ${ACCESS_TOKEN}" "https://$GEN3_ENDPOINT/manifests/metadata")
 
-    while [ -z "$MANIFEST" ] && [ -z "$METADATA_FILE" ]; do
-        if [ -z "$MANIFEST" ]; then
-            log "Unable to get manifests from '$GEN3_ENDPOINT/manifests'"
-            log $MANIFEST
+    while [ -z "$MANIFESTS" ] && [ -z "$METADATA" ]; do
+        if [ -z "$MANIFESTS" ]; then
+            log "Unable to get manifests from '$GEN3_ENDPOINT/manifests/'"
```

Comment:
Nitpick since its a log message but same question as the other comments I've left about the endpoint path